docs/src/pages/system/basics/basics.md

```diff
@@ -33,318 +23,339 @@ npm install @material-ui/system
+            laptop: 300,
+            desktop: 500,
+          },
+        }}
+      >
+        This box has a responsive width
+      </Box>
+    </ThemeProvider>
+  );
+}
+```
 
-const palette = compose(textColor, bgcolor);
+If you are using TypeScript, you will also need to use [module augmentation](/guides/typescript/#customization-of-theme) for the theme to accept the above values.
+
+```ts
+declare module '@material-ui/core/styles/createBreakpoints' {
+  interface BreakpointOverrides {
+    xs: false; // removes the `xs` breakpoint
+    sm: false;
+    md: false;
+    lg: false;
+    xl: false;
+    tablet: true; // adds the `tablet` breakpoint
+    laptop: true;
+    desktop: true;
+  }
+}
 ```
 
-## Variants
+### Theme getter
+
+If you wish to use the theme for a CSS property that is not supported natively by the system, you can use a function as the value, in which you can access the theme object.
 
-The `style()` helper can also be used to maps props to style objects in a theme.
-In this example, the `variant` prop supports all the keys present in `theme.typography`.
+{{"demo": "pages/system/basics/ValueAsFunction.js"}}
 
-{{"demo": "pages/system/basics/Variant.js", "defaultCodeOpen": true}}
+## Implementations
 
-## CSS prop
+The `sx` prop can be used in four different locations:
 
-If you want to support custom CSS values, you can use the `css()` helper.
-It will process the `css` prop.
+### 1. Core components
 
-{{"demo": "pages/system/basics/CssProp.js", "defaultCodeOpen": true}}
+All core Material-UI components will support the `sx` prop.
 
-## How it works
+### 2. Box
 
-styled-system has done a great job at [explaining how it works](https://github.com/jxnblk/styled-system/blob/master/docs/how-it-works.md#how-it-works).
-It can help building a mental model for this "style function" concept.
+[`Box`](/components/box/) is a lightweight component that gives access to the `sx` prop, and can be used as a utility component, and as a wrapper for other components.
+It renders a `<div>` element by default.
 
-## Real-world use case
+### 3. Custom components
 
-In practice, a Box component can save you a lot of time.
-In this example, we demonstrate how to reproduce a Banner component.
+In addition to Material-UI components, you can add the `sx` prop to your custom components too, by using the `experimentalStyled` utility from `@material-ui/core/styles`.
 
-{{"demo": "pages/system/basics/RealWorld.js", "bg": true}}
+```jsx
+import { experimentalStyled as styled } from '@material-ui/core/styles';
+
+const Div = styled('div')``;
+```
 
-## Prior art
+> **Note:**
+>
+> You should use this prop whenever you need to add or override a component style. If you find you are repeatedly applying the same styles to a component, then `styled()` may be a better option, as it allows you to specify the styles only once, and reuse them in all component instances. See [Customizing components](/customization/components/) for all the alternatives.
```

Comment:
(Note that that the _Customizing components_ section doesn't used `styled()`...

Reply:
[…] yet :)

Reply:
Yep, just sayin' :)